oonib/oonibackend.py

```diff
@@ -64,9 +64,9 @@ def getEndpoint(endpoint_config):
 
 def createService(endpoint, role, endpoint_config):
     if role == 'bouncer':
-        factory = ooniBouncer
+        factory = OONIBouncer()
```

Comment:
Good call, though I realised while refactoring this part that there was a redundant check for `config.main.bouncer_file` since it's existence is already tested down here: https://github.com/TheTorProject/ooni-backend/pull/91/files/89e2e4628198bf6f9cb13f0732c774ca0652324f#diff-47300978dd2c3f736ff16c63290517aaR157.

I thought a bit about what the possible consequences for only trapping it there are and I didn't see any case where it would be a problem. Though it's perhaps useful to still think of some possible edge case I may not have thought of.